src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Modal.css';
+
+function getHeader(onClose, title, headerContent) {
+  return (
+    <div className="modal__header">
+      <div className="modal__left--corner">
+        <button className="modal__close-button" type="button" onClick={ onClose }>
+          <i className="far fa-times" />
+        </button>
+      </div>
+      <div className="modal__center">
+        <h1 className="modal__title" data-testid="modal-title">{ title }</h1>
+      </div>
+      <div className="modal__right--corner">{ headerContent }</div>
+    </div>
+  );
+}
+
+function getBody(bodyContent) {
+  return (
+    <div className="modal__body">
+      { bodyContent }
+    </div>
+  );
+}
+
+function getFooter(footerContent) {
+  return (
+    !!footerContent && (
+      <div className="modal__footer">
+        { footerContent }
+      </div>
+    )
+  );
+}
```

Comment:
Maybe you discussed with @luism-s about this? But I don't see why these are written as accessor functions (getters)